lib/quantum/parser.ex

```diff
@@ -2,15 +2,15 @@ defmodule Quantum.Parser do
 
   @moduledoc false
 
-  def parse("*/" <> i, min, max) do
```

Comment:
Out of curiosity, what was the warning here?  It seems like `min/max` is a bit more descriptive than magic variable names